docs/roman/datamodels/new_model.rst

```diff
@@ -82,7 +70,7 @@ type "object", and should include the core schema:
 .. code-block:: yaml
 
   allOf:
-     - $ref: "http://roman.stsci.edu/schemas/core.schema.yaml"
+     - $ref: "http://romancal.stsci.edu/schemas/core.schema.yaml"
```

Comment:
What are these yaml files? The core schema id is: "http://stsci.edu/schemas/roman_datamodel/core.schema"

Reply:
That line in the rst file was merely copied from the JWST version, which I then 'Romanticized'.